src/views/datasets/ri-buoy/summary.vue

```diff
@@ -22,6 +54,8 @@
       The historical data available on this site has been compiled from the
       <a
         href="http://www.dem.ri.gov/programs/emergencyresponse/bart/stations.php"
+        target="_blank"
+        rel="noreferrer noopener"
         >Narragansett Bay Fixed-Site Monitoring Network</a
       >. See <a href="nbfsmn_disclaimer.pdf">the disclaimer</a> for more
```

Comment:
This link doesn't seem to work in the localhost version. Is the pdf only included in the deployed version?

Reply:
just opened a PR to fix this #33

Reply:
thank you!